props/prop_base.h

```diff
@@ -53,6 +53,51 @@ class SavePresetStateFile : public ConfigFile {
 #endif    
 };
 
+  struct SoundToPlay {
```

Comment:
Looks like it is indented two steps? (It shouldn't be)
Could be just github showing it to me weird.

Reply:
fixed